tests/browser/http/test_content_disposition.py

```diff
@@ -122,17 +126,17 @@ def test_inlwithasciifilenamepdf(self):
         with the latest Acrobat Reader plugin, or, in the case of Chrome, using
         the built-in PDF handler).
         """
-        self._check_filename('inline; filename="foo.pdf"', "foo.pdf")
-
+        header_checker.check_filename('inline; filename="foo.pdf"', "foo.pdf",
+                                      expected_inline=True)
 
-class AttachmentTests(AttachmentTestCase):
 
+class TestAttachment:
```

Comment:
-1 - note qutebrowser is python3 only where inheriting from `object` isn't needed, and throughout the codebase also isn't done - so we should keep it that way for consistency :smile: